enhancements/network/upgrades-with-ovn-ic.md

```diff
@@ -0,0 +1,245 @@
+
+#### Notes for hypershift
+The upgrade to IC for hypershift follows exactly the same path as outlined above for standalone openshift.
+
+There is however an extra key component that is used until 4.13 in hypershift: a route to sbdb that connects 4.13 nodes to 4.13 masters, specifically to the sbdb in master nodes. This is not relevant anymore in IC multizone, since every node will have its own local instance of sbdb.
+In order to prevent disruptions and always have a fully functioning cluster all throughout the upgrade to IC multizone, the route to sbdb needs to remain in place until all nodes have transitioned to IC multizone (end of phase 2). 
+This is why `008-route.yaml` is found also in the `multi-zone-interconnect-tmp` folder and is only removed along with ovnkube-master when we finally switch to the `multi-zone-interconnect` folder, when the upgrade ends.
+
+```text
+$ ls -l bindata/network/ovn-kubernetes/managed/single-zone-interconnect/
+-rw-rw-r-- 1    863 Aug 17 15:25  005-service.yaml
+-rw-rw-r-- 1    822 Aug 17 15:25  008-route.yaml
+-rw-rw-r-- 1  21941 Aug 17 15:25  alert-rules-control-plane.yaml
+-rw-rw-r-- 1   1976 Aug 17 15:25  monitor-master.yaml
+-rw-rw-r-- 1  43906 Aug 17 15:25  ovnkube-master.yaml
+-rw-rw-r-- 1  30036 Aug 17 15:25  ovnkube-node.yaml
+```
+
+```text
+$ ls -l bindata/network/ovn-kubernetes/managed/multi-zone-interconnect-tmp/
+lrwxrwxrwx 1    44 Aug 17 15:25 005-service.yaml -> ../single-zone-interconnect/005-service.yaml
+lrwxrwxrwx 1    42 Aug 17 15:25 008-route.yaml -> ../single-zone-interconnect/008-route.yaml
+-rw-rw-r-- 1  7234 Aug 17 15:25 alert-rules-control-plane.yaml
+lrwxrwxrwx 1    53 Aug 17 15:25 monitor-control-plane.yaml -> ../multi-zone-interconnect/monitor-control-plane.yaml
+lrwxrwxrwx 1    47 Aug 17 15:25 monitor-master.yaml -> ../single-zone-interconnect/monitor-master.yaml
+lrwxrwxrwx 1    53 Aug 17 15:25 ovnkube-control-plane.yaml -> ../multi-zone-interconnect/ovnkube-control-plane.yaml
+lrwxrwxrwx 1    47 Aug 17 15:25 ovnkube-master.yaml -> ../single-zone-interconnect/ovnkube-master.yaml
+lrwxrwxrwx 1    44 Aug 17 15:25 ovnkube-node.yaml -> ../multi-zone-interconnect/ovnkube-node.yaml
+```
+
+```text
+$ ls -l bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/
+-rw-rw-r-- 1   7163 Aug 17 15:25 alert-rules-control-plane.yaml
+-rw-rw-r-- 1   1862 Aug 17 15:25 monitor-control-plane.yaml
+-rw-rw-r-- 1   9959 Aug 17 15:25 ovnkube-control-plane.yaml
+-rw-rw-r-- 1  41563 Aug 28 11:18 ovnkube-node.yaml
+```
+
+### Openshift 4.14 fresh install
+A fresh install of Openshift 4.14 will start directly with IC multizone. CNO will apply the YAMLs from `bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/`
+
+
+### Test Plan
+CI upgrade jobs from 4.13 to 4.14 on all cloud platforms, with special attention to disruption tests, will validate the new upgrade path for OVN interconnect.
```

Comment:
what about non-cloud? :D we did test vsphere/bm though there are no disruption tests there huh? :) , let's just say all supported platforms on CI/CD pipeline?